libres/lib/enkf/obs_data.cpp

```diff
@@ -274,8 +273,8 @@ static void obs_block_initR(const obs_block_type *obs_block, matrix_type *R,
         matrix_free(obs_block->error_covar);
 }
 
-static void obs_block_initE(const obs_block_type *obs_block, matrix_type *E,
-                            const double *pert_var, int *__obs_offset) {
+void obs_block_initE(const obs_block_type *obs_block, matrix_type *E,
```

Comment:
The details of the `obs_block` was not originally meant to be an internal implementation detail of `obs_data` - I would recommend doing the testing fully through the `obs_data` api.

Reply:
Do you mean that I should not test the `obs_block_initE` function directly, but instead just test `obs_data_allocE`?

Reply:
I would do that.

Reply:
That makes sense, thanks!
I've pushed a fix.